### DISCUSSION TITLE
Check if length is list > than  if another object is true

### DISCUSSION THREAD

**Discussion by subramanian0331** [cue-lang/cue#2069] [Q&A] [answered]
```
Ha_enabled: bool
Nodes: [...#Nodes]

#Nodes{
 cpu: int
}
```

i want to check if Length of `Nodes` is `>3` if `Ha_enabled` is true.. else just use as is

**Comment by slewiskelly** [comment, marked as answer]
There are several ways, but this is probably the most readable:

```cue
import "list"

#Node: {
    cpu: int
}

HAEnabled: bool

Nodes: [...#Node]

if HAEnabled {
    Nodes: list.MinItems(3)
}
```

Also, FYI, [this](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) is how code can be formatted, when providing examples.

**Reply by myitcv** [reply]
Thanks @slewiskelly. 

@subramanian0331 welcome to the project. I updated the formatting in your example to make it more readable.
